base/latex209.dtx

```diff
@@ -29,7 +29,7 @@
 %
 % \title{Compatibility mode for \LaTeXe{} emulating \LaTeX~2.09}
 % \author{Alan Jeffrey and Frank Mittelbach}
-% \date{1995/12/27}
+% \date{2024/11/16}
 %
```

Comment:
We should have a screenshot of this in the news :)

Reply:
feel free :-)  but it is really not a bug fix after nearly 30 years but just due to a mechanism change

Reply:
There was a smiley - and I didn't mean as a bug fix, rather as showing a _long_ term commitment to keeping compatibility working

Reply:
perhaps a nice story on the commitment to compatiblity, indeed ... case to write a note?